### PR TITLE
Replace the `participating` position with `important`, `worth prototyping`, and `non-harmful`

### LINKED ISSUE #51: Set of possible positions seems too narrow
At least for PRs against WHATWG standards we need a stronger signal than "participating" gives. We need something that indicates support. That's not a commitment to implement, but it is an endorsement of the feature that allows others to go ahead and implement it, to add tests to web-platform-tests, etc.

My initial hope for this repository was that we could have something where one could raise a standards-related issue with Mozilla stakeholders and determine a position on that issue. The way it's structured now doesn't really lend itself to that.

### PR DESCRIPTION
**Pull request by dbaron**
Fixes #51.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ Implementation status (or even intention) isn't tracked here; see [Firefox Platf
 The currently possible positions are:
 
 - `under consideration` - Mozilla's position on this specification is being discussed.
```
Comment (jyasskin) [inline README.md]: This might belong between `non-harmful` and `defer`, so that the positions are sorted by how favorable Mozilla is.

Comment (dbaron) [reply README.md]: I think I wanted to keep the current status of having `under consideration` at the start, since it's actually *not* a position.

Comment (jyasskin) [reply README.md]: Fair enough.

**Comment by ekr** [comment]
LGTM

**Pull request merged**